src/workloads/query/TimeseriesStressUnpacking.yml

```diff
@@ -0,0 +1,276 @@
+        Predicate: { $lte: 9880 }
+        FieldsToProject: *Project1Field
+
+  - ActorFromTemplate:
+      TemplateName: RunMatchProjectPipeline
+      TemplateParameters:
+        Name: "match_10_buckets_include_10_fields"
+        Active: [5]
+        # With 988 documents per bucket, 10 buckets will have 'measurement' <= 9880 (988 * 10).
+        Predicate: { $lte: 9880 }
+        FieldsToProject: *Project10Fields
+
+  - ActorFromTemplate:
+      TemplateName: RunMatchPipeline
+      TemplateParameters:
+        Name: "match_10_buckets_exclude"
+        Active: [6]
+        # With 988 documents per bucket, 10 buckets will have 'measurement' <= 9880 (988 * 10).
+        Predicate: { $lte: 9880 }
+
+  - ActorFromTemplate:
+      TemplateName: RunMatchProjectPipeline
+      TemplateParameters:
+        Name: "match_100_buckets_include_1_field"
+        Active: [7]
+        # With 988 documents per bucket, 100 buckets will have 'measurement' <= 98800 (988 * 100).
+        Predicate: { $lte: 98800 }
+        FieldsToProject: *Project1Field
+
+  - ActorFromTemplate:
+      TemplateName: RunMatchProjectPipeline
+      TemplateParameters:
+        Name: "match_100_buckets_include_10_fields"
+        Active: [8]
+        # With 988 documents per bucket, 100 buckets will have 'measurement' <= 98800 (988 * 100).
+        Predicate: { $lte: 98800 }
+        FieldsToProject: *Project10Fields
+
+  - ActorFromTemplate:
+      TemplateName: RunMatchPipeline
+      TemplateParameters:
+        Name: "match_100_buckets_exclude"
+        Active: [9]
+        # With 988 documents per bucket, 100 buckets will have 'measurement' <= 98800 (988 * 100).
+        Predicate: { $lte: 98800 }
+        # Without a $project stage, $_internalUnpackBucket will have an "exclude" parameter.
+
+  - ActorFromTemplate:
+      TemplateName: RunMatchProjectPipeline
+      TemplateParameters:
+        Name: "match_1000_buckets_include_1_field"
+        Active: [10]
+        # With 988 documents per bucket, 1000 buckets will have 'measurement' <= 988000 (988 * 1000).
+        Predicate: { $lte: 988000 }
+        FieldsToProject: *Project1Field
+
+  - ActorFromTemplate:
+      TemplateName: RunMatchProjectPipeline
+      TemplateParameters:
+        Name: "match_1000_buckets_include_10_fields"
+        Active: [11]
+        # With 988 documents per bucket, 1000 buckets will have 'measurement' <= 988000 (988 * 1000).
+        Predicate: { $lte: 988000 }
+        FieldsToProject: *Project10Fields
+
+  - ActorFromTemplate:
+      TemplateName: RunMatchPipeline
+      TemplateParameters:
+        Name: "match_1000_buckets_exclude"
+        Active: [12]
+        # With 988 documents per bucket, 1000 buckets will have 'measurement' <= 988000 (988 * 1000).
+        Predicate: { $lte: 988000 }
+        # Without a $project stage, $_internalUnpackBucket will have an "exclude" parameter.
+
+AutoRun:
+  - When:
+      mongodb_setup:
+        $eq:
+          - standalone-all-feature-flags
```

Comment:
I think we should use the same variants as all the other Time-series workloads. The `replica` and `replica-all-feature-flags`. Makes it easier to do comparisons.

https://github.com/mongodb/genny/blob/2f11236c169a8a5abf2b6b38202340057276b665/src/workloads/query/TimeseriesCount.yml#L124-L127